lib/rollup/index.js

```diff
@@ -13,13 +13,11 @@ const plugin = (config) => async (files, metalsmith, done) => {
   const file = normalize(config.input)
   const map = `${file}.map`
 
-  // Resolve paths from Metalsmith source
-  config.input = resolve(metalsmith.source(), file)
```

Comment:
We no longer overwrite `config.input` so paths stay as:

```console
'javascripts/head-ie8.js'
'javascripts/application.js'
'javascripts/application-ie8.js'
'javascripts/govuk-frontend.js'
'javascripts/example.js'
```

They remain as child paths to both `metalsmith.source()` and `metalsmith.destination()`